sdk/core/Azure.Core/src/Shared/Argument.cs

```diff
@@ -179,5 +179,38 @@ public static void AssertNotNullOrWhiteSpace(string value, string name)
                 throw new ArgumentOutOfRangeException(name, "Value is greater than the maximum allowed.");
             }
         }
+
+        /// <summary>
+        /// Throws if <paramref name="value"/> has not been initialized, otherwise return <paramref name="value"/>.
```

Comment:
Nit: 
```suggestion
        /// Throws if <paramref name="value"/> has not been initialized; otherwise, returns <paramref name="value"/>.
```

Same below.